src/api/functions.ts

```diff
@@ -145,11 +151,12 @@ async function createFunctionVersion(
     form.append('Visibility', fn.access);
     form.append('Content', fn.content, contentOpts);
 
-    const resp = await client.post(
+    const resp = await client.requestText(
```

Comment:
Do we need separate methods for this?

Reply:
This was an interesting issue with types that I had. When I defined the `request` method with the options type:

```typescript
options: OptionsOfJSONResponseBody | OptionsOfTextResponseBody
```

The return type signature of the function completely changed and messed everything up. I was out of patience with types at the time and so I created a new function. I would have preferred for the `request` function to work here. Maybe we can pair on some type things here?

Reply:
Hmmm I'm fine with it but we are not really doing anything different in it. So from that perspective we could  also just type it with the Got Options rather than the specific type of Options

Reply:
Oh, OK. I'll see what that does to the types. Hope it helps! 😄